Technical-Steering-Committee/coordinated-disclosure-details.md

```diff
@@ -0,0 +1,43 @@
+# What does the Zowe Disclosure process look like?
+
+There are two main processes to obtain information about security vulnerabilities. One process is used by
+external stakeholders, and the other process is used by internal stakeholders. 
+
+## Step 1: New issue reported
+
+An external stakeholder reports issues via the zowe-security@openmainframeproject.org mailing list. 
+This issue is then discussed at the weekly Security Workgroup call. At this time, issues within
+the security-reports (private) Repository are created to track related information. 
+
+Alternatively, external stakeholders can report their findings through a CVE Numbering Authorities (CNA).
+In this case, the Zowe organization will be notified, as members of the Federated CVE program, and subsequently can discuss
+and agree on what to do with the vulnerability. If needed, the Zowe organization will engage the author in discussions around
+coordinated vulnerability disclosure. 
+
+The internal stakeholders report the issue to the security workgroup through a security 
+workgroup member. The issue in the security-reports (private) repository is created to track related 
+information. 
+
+## Step 2: Impact assessment
+
+The first step after the security workgroup is informed about the vulnerability is to verify the assessment
+already done by the party bringing the vulnerability forward. A Security workgroup member will
+be assigned to the issue for further investigation and will track the issue until closure. 
```

Comment:
I might be missing the part when we actually publish the fix before disclosing the vulnerability.
I assume that's done as part of the closure process?

Reply:
Clarified.

Reply:
Thank you!